WordPress/src/main/java/org/wordpress/android/ui/posts/PostListAction.kt

```diff
@@ -91,5 +101,16 @@ fun handlePostListAction(
         is PostListAction.DismissPendingNotification -> {
             NativeNotificationsUtils.dismissNotification(action.pushId, activity)
         }
+        is PostListAction.CopyUrl -> {
+            try {
+                activity.clipboardManager!!.setPrimaryClip(
```

Comment:
This seems like an edge case but what do you think of adding an optional call instead of a force unwrap on this and handle the case where the `clipboardManager` is null accordingly?

Reply:
Very good suggestion, I'm more in favor of it than what's currently in the code 🙇🏻‍♂️

Reply:
Applied the suggestion and throwing an own `NPE` if clipboard manager was null.
Please review!

Reply:
Thank you for applying the suggestion with 60c19e5eea0343ebe3b80446484f1a5eea3007bf. LGTM :)